phpBB/phpbb/di/container_builder.php

```diff
@@ -135,6 +135,11 @@ public function get_container()
 			$config_cache = new ConfigCache($container_filename, defined('DEBUG'));
 			if ($this->use_cache && $config_cache->isFresh())
 			{
+				if ($this->use_extensions)
+				{
+					require $this->get_autoload_filename();
```

Comment:
We usually use ```require(...)```. Also see the piece of code right below this. I suggest we use require() everywhere.